pkg/runtime/options.go

```diff
@@ -19,6 +20,7 @@ type (
 		configurations  []configuration.Configuration
 		locks           []lock.Lock
 		pubsubs         []pubsub.PubSub
+		workFlows       []workflows.Workflow
```

Comment:
nit: "workflow" is one word, so we should use all lowercase letters in our "workflow" variable and function names.